classes/controller/AdminController.php

```diff
@@ -2629,6 +2630,7 @@ public function setMedia($isNewTheme = false)
             $this->addCSS(__PS_BASE_URI__ . $this->admin_webpath . '/themes/' . $this->bo_theme . '/css/' . $this->bo_css, 'all', 0);
             $this->addCSS(__PS_BASE_URI__ . $this->admin_webpath . '/themes/' . $this->bo_theme . '/css/vendor/titatoggle-min.css', 'all', 0);
             $this->addCSS(__PS_BASE_URI__ . $this->admin_webpath . '/themes/' . $this->bo_theme . '/public/theme.css', 'all', 0);
+            $this->addCSS(__PS_BASE_URI__ . $this->admin_webpath . '/themes/new-theme/public/create_product_default_theme.css', 'all', 0);
```

Comment:
Actually, it seems like this css needs to be added in both default AND new-theme.

I tested the modal on the product page and it displays fine because the product CSS has details for this modal, it also works fine in a legacy page thanks to this `create_product_default_theme.css`. However, when you test it on a migrated page it doesn't have the product CSS nor the modal CSS and it is displayed way too wide:

![Capture d’écran 2022-09-07 à 19 51 47](https://user-images.githubusercontent.com/13801017/188946119-8042f78f-0504-4ecf-a3d7-040b0452b554.png)

So you should move this `addCss` call outside of the if/else block so that it's added on every page.